### PR TITLE
Add postgres and rds-postgres examples

### PR DESCRIPTION
**Pull request by fabianvf**

### CODE REVIEW

File: postgresql-apb/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+* Deploys 1 postgresql.
+
+## Requirements
+
+## Parameters
+* NAMESPACE: Optional, default 'postgresql-apb', Namespace to deploy the cluster in.
+* POSTGRESQL_DATABASE, default 'admin', Postgresql database name.
+* POSTGRESQL_PASSWORD, default 'admin', Postgresql database password.
+* POSTGRESQL_USER, default 'admin', Postgresql database username.
+* OPENSHIFT_TARTET: Required, target openshift deployment
```
Comment (cfchase) [inline postgresql-apb/README.md]: (minor spelling) OPENSHIFT_TAR**G**ET

File: postgresql-demo-apb/apb.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+id: 26355d1a-0301-4841-85af-536a4d3afaa9
+name: ansibleplaybookbundle/postgresql-demo-apb
```
Comment (jmontleon) [inline postgresql-demo-apb/apb.yml]: To make this work in the web UI my understanding is you the name needs to be split:
```
name: postgresql-demo-apb  
image: ansibleplaybookbundle/postgresql-demo-apb  
```
And you also need at least:
```
tags:
 - database
metadata:
  displayName: ""
  longDescription: ""
  imageUrl: "<url for png or svg>"
  documentationUrl: ""
```

**Review by jmontleon — CHANGES_REQUESTED** [review]

File: postgresql-demo-apb/apb.yml (hunk 2 above)
Comment (fabianvf) [reply postgresql-demo-apb/apb.yml]: Good to know, is imageUrl required?

**Review by jmontleon — APPROVED** [review]

**Comment by jmontleon** [comment]
ACK

**Pull request merged**